### PR TITLE
feat: change name and symbol

### PR DESCRIPTION
**Pull request by neanvo**

### CODE REVIEW

File: projects/liquid-restaking/contracts/cToken.sol [hunk 1]
```diff
@@ -45,10 +48,13 @@ contract cToken is Configurable, ERC20PausableUpgradeable, ICToken {
         __ERC20_init(name, symbol);
         __Pausable_init();
         __ERC20Pausable_init();
-        __cToken_init();
+        __cToken_init(name, symbol);
     }
 
-    function __cToken_init() internal {}
+    function __cToken_init(string memory name, string memory symbol) internal {
+        _name = name;
```
Comment (intosKai) [inline projects/liquid-restaking/contracts/cToken.sol:55]:
```suggestion
        _changeName(name);
```

File: projects/liquid-restaking/contracts/cToken.sol [hunk 2]
```diff
@@ -45,10 +48,13 @@ contract cToken is Configurable, ERC20PausableUpgradeable, ICToken {
...
+        _symbol = symbol;
```
Comment (intosKai) [inline projects/liquid-restaking/contracts/cToken.sol:56]:
```suggestion
        _changeSymbol(symbol);
```

File: projects/liquid-restaking/contracts/cToken.sol [hunk 3]
```diff
@@ -138,4 +144,57 @@ contract cToken is Configurable, ERC20PausableUpgradeable, ICToken {
...
+     */
+    function _changeName(string memory newName) internal {
+        _name = newName;
+    }
+
+    /**
+     * @dev Internal function to change the symbol of the token.
+     */
+    function _changeSymbol(string memory newSymbol) internal {
+        _symbol = newSymbol;
```
Comment (intosKai) [inline projects/liquid-restaking/contracts/cToken.sol:179]: emit Event about update

File: projects/liquid-restaking/contracts/cToken.sol [hunk 4]
```diff
@@ -138,4 +144,57 @@ contract cToken is Configurable, ERC20PausableUpgradeable, ICToken {
...
+
+    /*******************************************************************************
+                        INTERNAL FUNCTIONS
+    *******************************************************************************/
+
+    /**
+     * @dev Internal function to change the name of the token.
+     */
+    function _changeName(string memory newName) internal {
+        _name = newName;
```
Comment (intosKai) [inline projects/liquid-restaking/contracts/cToken.sol:172]: emit Event about update

File: projects/liquid-restaking/contracts/cToken.sol [hunk 5]
```diff
@@ -19,6 +19,9 @@ import "./interfaces/ICToken.sol";
 contract cToken is Configurable, ERC20PausableUpgradeable, ICToken {
     using Math for uint256;
 
+    string private _name;
+    string private _symbol;
+
     /**
      * @dev This empty reserved space is put in place to allow future versions to add new
      * variables without shifting down storage in the inheritance chain.
```
Comment (intosKai) [inline projects/liquid-restaking/contracts/cToken.sol:27]: decrease gap

File: projects/rate-provider/scripts/deploy/rate-provider.ts [hunk 6]
```diff
@@ -1,30 +1,30 @@
...
-    const args = ['0xf073bAC22DAb7FaF4a3Dd6c6189a70D54110525C']; // genETH
+  const args = ["0xf073bAC22DAb7FaF4a3Dd6c6189a70D54110525C"]; // InETH
 
-    const rateProvider = await ethers.deployContract(
-        'GenEthRateProvider',
-        args
-    );
-    await rateProvider.waitForDeployment();
+  const rateProvider = await ethers.deployContract(
+    "GenEthRateProvider",
```
Comment (intosKai) [inline projects/rate-provider/scripts/deploy/rate-provider.ts:9]:
```suggestion
    "InEthRateProvider",
```

File: projects/rate-provider/scripts/deploy/rate-provider.ts [hunk 7]
```diff
@@ -1,30 +1,30 @@
...
 
-    await run('verify:verify', {
-        address: await rateProvider.getAddress(),
-        constructorArguments: args,
-        contract: 'contracts/RateProvider.sol:GenEthRateProvider',
-    });
+  await run("verify:verify", {
+    address: await rateProvider.getAddress(),
+    constructorArguments: args,
+    contract: "contracts/RateProvider.sol:GenEthRateProvider",
```
Comment (intosKai) [inline projects/rate-provider/scripts/deploy/rate-provider.ts:23]:
```suggestion
    contract: "contracts/RateProvider.sol:InEthRateProvider",
```

**Review by intosKai — APPROVED** [review]

**Review by intosKai — APPROVED** [review]

**Review by infbque — APPROVED** [review]

**Pull request merged**